Sources/Positionless/Algorithms/SinglePass.swift

```diff
@@ -6,7 +6,7 @@ extension Collection {
   /// `true`.
   @discardableResult
   func forEachUntil(_ op: (borrowing Element) -> Bool) -> Bool {
```

Comment:
```suggestion
  func forEach(until: op: (borrowing Element) -> Bool) -> Bool {
```
(Per API guidelines)